src/main/java/org/sagebionetworks/web/client/jsinterop/React.java

```diff
@@ -12,6 +12,19 @@ public class React {
 
 	public static native <P extends ReactComponentProps> ReactNode createElement(ReactComponentType<P> component, P props, ReactNode[] children);
 
+	public static native <P extends ReactComponentProps> ReactNode createElement(ReactComponentType<P> component, P props, ReactNode child1, ReactNode child2);
+
+	/**
+	 * Similar to {@link #createElementWithSynapseContext} but only includes the theme. Any components rendered will NOT get the full Synapse context, including
+	 * access token + auth state, experimental mode status, and time display settings.
+	 */
+	@JsOverlay
+	public static <P extends ReactComponentProps> ReactNode createElementWithThemeContext(ReactComponentType<P> component, P props) {
+		SynapseContextProviderProps emptyContext = SynapseContextProviderProps.create(SynapseContextJsObject.create(null, false, false), null);
+		return createElementWithSynapseContext(component, props, emptyContext);
+	}
```

Comment:
Since our MUI components now use the theme context instead of inline styles, components that didn't get context lost their styles.

One example of this is FullWidthAlert. The FullWidthAlert widget cannot have the context props injected by Gin without a large refactor because we use it in UI XML.

To ensure it gets wrapped in theme context, we use this overlay method instead of plain React.createElement.